packages/renderer/src/lib/deployments/DeploymentsList.svelte

```diff
@@ -1,6 +1,13 @@
 <script lang="ts">
 import { faTrash } from '@fortawesome/free-solid-svg-icons';
-import { Button } from '@podman-desktop/ui-svelte';
+import {
+  Button,
+  Table,
+  TableColumn,
+  TableDurationColumn,
+  TableRow,
+  TableSimpleColumn,
```

Comment:
Not related to this PR, but I think it might be nicer to reduce the number of imports with https://github.com/containers/podman-desktop/issues/7189